src/App.js

```diff
@@ -1,12 +1,24 @@
-import logo from './logo.svg';
+import { useDispatch } from 'react-redux';
+import React, { useEffect } from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { fetchCountriesData } from './Redux/covidData/countriesCovidData';
+import HomePage from './components/home/Home';
+import Details from './components/details/Details';
 import './App.css';
 
-function App() {
+const App = () => {
```

Comment:
The styling of the list on your Homepage does not conform to the alternating dual-color pattern of the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), and therefore difficult to differentiate the details of each list item. Kindly note we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that:
- [ ] The cards/ cells on the _Home page_ are **two columns** on a mobile screen.
- [ ] The cells/ card of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (alternating) background color(s)**.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/164912547-dca62494-1d11-4a95-b3a0-362be1de7b81.png)



**The original design:**
![image](https://user-images.githubusercontent.com/12745474/142270463-4d16b9d3-7557-430c-80eb-a7c40589a4a6.png)